tsconfig.esm5.json

```diff
@@ -1,7 +1,7 @@
 {
 	"extends": "./tsconfig.json",
 	"compilerOptions": {
-		"module": "es2015",
+		"module": "es2020",
```

Comment:
needed to support dynamic imports, IMHO no much sense to keep es2015 anymore as all our UI projects does not support old browser versions anymore:

https://github.com/balena-io/balena-ui/blob/d05abdcabcc0fc1d4cd41636a305d7ab84fface8/tsconfig.json#L3

Reply:
I agree, but this should become a major just to be picky. We could even maybe decide to start expiring only in es2020 from now on.
Perhaps it's an opportunity to bump the `target` to at least `es2015`, like in the UI.